### PR TITLE
cityworks updates

### PR DESCRIPTION
**Pull request by mottesen-cityworks**

### CODE REVIEW

File: CityworksConnection/Connect2Cityworks.pyt [hunk 1]
```diff
@@ -351,6 +351,7 @@ class Tool(object):
                 webmap = WebMap(mapitem)
 
                 for layer in webmap['operationalLayers']:
+                for layer in webmap.layers:
```
Comment (allisonmuise) [inline CityworksConnection/Connect2Cityworks.pyt]: either line 353 or line 354 must be deleted, or else the indentations need to be fixed.

Comment (allisonmuise) [reply CityworksConnection/Connect2Cityworks.pyt]: @mottesen-cityworks

Comment (allisonmuise) [reply CityworksConnection/Connect2Cityworks.pyt]: @mottesen-cityworks Could you review/make this change so I can merge?

Comment (aparker-cityworks) [reply CityworksConnection/Connect2Cityworks.pyt]: This has already been fixed in https://github.com/Esri/crowdsource-reporter-scripts/pull/57
You can close https://github.com/Esri/crowdsource-reporter-scripts/pull/53 it's not needed anymore.

**Pull request merged**